apps/src/aichat/views/modelCustomization/CompareModelsDialog.tsx

```diff
@@ -1,17 +1,29 @@
 import React, {useState} from 'react';
 
+import {useAppSelector} from '@cdo/apps/util/reduxHooks';
 import AccessibleDialog from '@cdo/apps/templates/AccessibleDialog';
 import {Heading3} from '@cdo/apps/componentLibrary/typography';
 import Button from '@cdo/apps/componentLibrary/button/Button';
 
 import ModelDescriptionPanel from './ModelDescriptionPanel';
 import styles from './compare-models-dialog.module.scss';
+import {ModelDescription} from '../../types';
 
-const CompareModelsDialog: React.FunctionComponent<{onClose: () => void}> = ({
-  onClose,
-}) => {
-  const [chosenModelLeft, setChosenModelLeft] = useState<string>('llama2');
-  const [chosenModelRight, setChosenModelRight] = useState<string>('mistral');
+const CompareModelsDialog: React.FunctionComponent<{
+  onClose: () => void;
+  availableModels: ModelDescription[];
```

Comment:
I think it makes sense to pass the selectedModel in here instead of getting it from redux?